deployment/docker-compose.test.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
```

Comment:
The proper way is to set environment variables, or `.env` file for testing, and then use the override file in `docker-compose.override.example.yml`. We should delete this file if possible.
We can use the file, but only if it extends the original `docker-compose.yml` not replace it.